splinepy/io/gismo.py

```diff
@@ -80,7 +80,7 @@ def _array_to_text(array, is_matrix, as_b64):
             field_mask = _np.arange(len(multipatch.fields))
         supports = _np.array(
             [
-                (i, j, 0)
+                (i, k, 0)
```

Comment:
I think we don't need the enumerate block here do we? J is unused

Reply:
was changed back again in a later commit for successful field mask